src/templates/form-fields-inputs/fieldSwitchBlock.vue

```diff
@@ -66,10 +66,16 @@
     name: {
       type: String,
       required: true
+    },
+    value: {
+      default: undefined,
+      type: Boolean,
+      required: false
```

Comment:
Booleano não precisa aplicar required. E default como "undefined" não faz sentido. Aplicar default apenas se for "true"

```suggestion
      type: Boolean,
```